geometry_msgs/PoseStamped approachPose
geometry_msgs/PoseStamped pivotPose